parm/namelist.mpassit

```diff
@@ -1,5 +1,5 @@
 &config
-    grid_file_input_grid = "history.@timestr@.nc"
+    grid_file_input_grid = "init.nc"
```

Comment:
Linking a file from COM to stmp is NOT allowed. We need to copy it.

I have two questions:
1. Can we use `static.nc`, `invariant.nc` or `mapsout.nc` in the same cycle for `grid_file_input_grid`? 
2. Any `init.nc` will work for this purpose? If yes, we may add init.nc as a fix files
3. Will a potential MPASSIT update resolve this crash?

Reply:
I have tried invariant.nc and it didn't work. We might need more input from @EricJames-NOAA

Reply:
Based on the offline conversation, we will further look at this outside of this PR.

Reply:
@chunhuazhou @EricJames-NOAA 
While we can use this temporary solution at this moment, it is good to add the missed `dzs` field into the MPAS-Model history output stream in `Registry.xml`.

For references, here is where `dzs` was added in MPASSIT:
https://github.com/NOAA-GSL/MPASSIT/blob/d517ec21796c9faecc1005a244db893364e44096/model_grid.F90#L400-L401

We can add it to the `Registry.xml` here:
https://github.com/NOAA-GSL/MPAS-Model/blob/fba865e9e82103aec6e316d809225c11d2581ee6/src/core_atmosphere/Registry.xml#L905